app/mailers/digest_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class DigestMailer < ActionMailer::Base
+  default from: "Crantastic <cranatic@gmail.com>"
+
+  def weekly_digest(digest)
+    @digest = digest
+    mail to: "r-help@r-project.org",
```

Comment:
I just moved the mailer, I didn't change the rake task that sends it (Which I don't think has been run since October?).